types/multi_address.go

```diff
@@ -100,6 +102,21 @@ func (m *MultiAddress) Decode(decoder scale.Decoder) error {
 		return err
 	}
 
+	if defaultOptions.NoPalletIndices {
```

Comment:
Similar question as above. Also, if there are no pallet indices, do we always get an `AccountID`?

Reply:
See my reply to the question above.